app/views/password/password_form.html.erb

```diff
@@ -4,20 +4,26 @@
 
 <h1 class="govuk-heading-l"><%= t 'password.change_password_heading' %></h1>
 
-<%= form_for @password_form, url: profile_change_password_path do |f| %>
+<%=error_summary_for(@password_form&.errors, :form) %>
+<%= form_for @password_form, url: profile_change_password_path, as: :form do |f| %>
```

Comment:
by adding the `as: :form` you're now sending the changes to the controller under the `form` params (`params[:form]`), but the controller expects the content be in  the `change_password_form` params (`params[:change_password_form]`)